packages/tracking_repository/lib/tracking_repository.dart

```diff
@@ -1,3 +1,4 @@
 library tracking_repository;
 
+export 'src/landing_page_events.dart';
```

Comment:
If I am not mistaken, there is two files called `tracking_repository.dart`. I think we should rename them to better distinguish them.

Reply:
That's standard naming as is a best practice in case you also expose an object with the same name as the package itself.